broker-daemon/bin/sparkswapd.js

```diff
@@ -5,97 +5,104 @@
-  RPC_PRIV_KEY,
-  RPC_PUB_KEY,
-  RPC_USER,
-  RPC_PASS
-} = process.env
+  throw new Error(`Invalid engine type: ${engineType}`)
+}
 
 // TODO: Add validations to ./bin/sparkswapd when they become available
 program
   .version(CLI_VERSION)
-  .option('--rpc-address [rpc-address]', 'Add a host/port to listen for daemon RPC connections', validations.isHost, RPC_ADDRESS)
-  .option('--interchain-router-address [server]', 'Add a host/port to listen for interchain router RPC connections', validations.isHost, INTERCHAIN_ROUTER_ADDRESS)
-  .option('--data-dir [path]', 'Location to store SparkSwap data', validations.isFormattedPath, DATA_DIR)
-  .option('--disable-auth [disable-auth]', 'Disable SSL for the broker (DEV ONLY)', program.BOOL, DISABLE_AUTH)
-  .option('--rpc-user [rpc-user]', 'Rpc user name, used when auth is enabled', program.String, RPC_USER)
-  .option('--rpc-pass [rpc-pass]', 'Rpc password, used when auth is enabled', program.String, RPC_PASS)
-  .option('--rpc-privkey-path [path>', 'Location of private key for the broker\'s rpc', validations.isFormattedPath, RPC_PRIV_KEY)
-  .option('--rpc-pubkey-path [path]', 'Location of the public key for the broker\'s rpc', validations.isFormattedPath, RPC_PUB_KEY)
-  .option('--disable-relayer-auth [disable-relayer-auth]', 'Disable SSL and message signing to the relayer (DEV ONLY)', program.BOOL, DISABLE_RELAYER_AUTH)
-  .option('--id-privkey-path [path]', 'Location of private key for the broker\'s identity', validations.isFormattedPath, ID_PRIV_KEY)
-  .option('--id-pubkey-path [path]', 'Location of the public key for the broker\'s identity', validations.isFormattedPath, ID_PUB_KEY)
-  .option('--markets [markets]', 'Comma-separated market names to track on startup', validations.areValidMarketNames, MARKETS)
-  .option('--relayer-host [server]', 'The host address for the SparkSwap Relayer', validations.isHost, RELAYER_RPC_HOST)
-  .option('--relayer-cert-path [path]', 'Location of the root certificate for the SparkSwap Relayer', validations.isFormattedPath, RELAYER_CERT_PATH)
+  // Broker configuration options
+  .option('--data-dir [data-dir]', 'Location to store SparkSwap data', validations.isFormattedPath, config.dataDir)
+  .option('--interchain-router-address [interchain-router-address]', 'Add a host/port to listen for interchain router RPC connections', validations.isHost, config.interchainRouterAddress)
+  .option('--id-pub-key [id-pub-key]', 'Location of the public key for the broker\'s identity', validations.isFormattedPath, config.idPubKey)
+  .option('--id-priv-key [id-priv-key]', 'Location of private key for the broker\'s identity', validations.isFormattedPath, config.idPrivKey)
+  // TODO: Make this a list instead of string
+  .option('--markets [markets]', 'Comma-separated market names to track on startup', validations.areValidMarketNames, config.markets)
+  .option('--disable-auth [disable-auth]', 'Disable SSL for the broker (DEV ONLY)', program.BOOL, config.disableAuth)
```

Comment:
This doesn't need an actual parameter does it? Is the right way to list this: `'--disable-auth'` instead of including `[disable-auth]`? (I know your PR didn't introduce this, just noticed it).